fintech-examples/fintech-ui/src/app/errorsHandler/info/info.service.ts

```diff
@@ -16,7 +16,7 @@ export class InfoService {
   private readonly DEFAULT_OPTIONS: InfoOptions = {
     severity: 'info',
     closable: true,
-    duration: 6000
+    duration: 60000
```

Comment:
Why do we need to increase duration in default  options?

Reply:
was not able to read message and did not know how to change otherwise. changed back to 6000. But how can this be changed otherwise?